experiments/model_development/train.py

```diff
@@ -9,4 +9,7 @@
 
 # Construct a training job and run it with per-epoch evaluation.
 job = TrainingJob(config=config)
```

Comment:
is train.py still how training is being done with the new models (e.g. Timesformer?)

Reply:
yes: the heavy lifting happens in utils/train_job.py, where most of the code remains unchanged except for the model used in TrainingJob